src/main/java/uk/gov/mca/beacons/service/model/BeaconStatus.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.mca.beacons.service.model;
+
+/**
+ * An enumeration of the possible states that a beacon registration can be in.
+ */
+public enum BeaconStatus {
+  NEW,
```

Comment:
This is taken straight from the Trello ticket here: https://trello.com/c/bZJIFyk5/209-accept-and-send-button-in-webform-saves-beacon-registration

Acceptance Criteria point 3